### PR TITLE
Buffs lunging gloves a bit, so they arent a detriment to the user as much

### PR DESCRIPTION
**Pull request by Arturlang**
## About The Pull Request
Buffs the leaping gloves all around, the hardstuns are neccesary so the victim cant just instantly squeeze out of grabs or crawl away, increased the stamina damage at places as well.

## Why It's Good For The Game
Leaping at the moment feels incredibly useless, and more a detriment to the user or anything, this hopes to change that.

## Changelog
:cl:
tweak: Leaping should be a bit more powerfull all around
/:cl:

This probably will need a bit of tweaking, but this PR is atleast worth a testmerge

### CODE REVIEW

**Comment by DeltaFire15** [comment]
I really don't think these need to be buffed. They're already very effective for bridging distances and knocking enemies down, even if just briefly. Leaving people the ability to get away from such an attack is neccessary aswell as we do not need more instant-win methods.

**Comment by Arturlang** [comment]
The gloves by themselves still wont win the fight for you, but it should atleast make them an actual security tool instead of a fun past-time

**Comment by Trilbyspaceclone** [comment]
This is just slip combat but with extra steps and gloves... - This is a joke.

**Comment by Arturlang** [comment]
Does it have to be? Lunging is a risky move, it should be  rewarding if done right as well, instead of producing negligable results.

**Comment by ghost** [comment]
just remember that if you miss your shot you have a very good chance of permanently removing yourself from the round

File: code/datums/components/tackle.dm [hunk 1]
```diff
@@ -192,9 +192,9 @@
 			user.SetKnockdown(0)
 			user.set_resting(FALSE, TRUE, FALSE)
 			user.forceMove(get_turf(target))
-			target.adjustStaminaLoss(40)
-			target.Paralyze(5)
-			target.Knockdown(30)
+			target.adjustStaminaLoss(50)
+			target.Paralyze(8) //Otherwise the victim can just instantly get out of the grab.
```
Comment (silicons) [inline code/datums/components/tackle.dm]: you are aware that this is MEANT to be a passive grab for a reason and not just being an instant upgrade to aggro, yes?

File: code/datums/components/tackle.dm [hunk 2]
```diff
@@ -163,24 +163,24 @@
...
 			user.Knockdown(20)
-			target.Knockdown(25)
+			target.Knockdown(30)
 
 		if(1 to 2) // solid hit, tackler has a slight advantage
 			user.visible_message("<span class='warning'>[user] lands a solid [tackle_word] on [target], knocking them both down hard!</span>", "<span class='userdanger'>You land a solid [tackle_word] on [target], knocking you both down hard!</span>", target)
 			to_chat(target, "<span class='userdanger'>[user] lands a solid [tackle_word] on you, knocking you both down hard!</span>")
 
-			target.adjustStaminaLoss(30)
+			target.adjustStaminaLoss(50)
```
Comment (silicons) [inline code/datums/components/tackle.dm]: 40

File: code/datums/components/tackle.dm [hunk 3]
```diff
@@ -163,24 +163,24 @@
...
 		if(-1 to 0) // decent hit, both parties are about equally inconvenienced
 			user.visible_message("<span class='warning'>[user] lands a passable [tackle_word] on [target], sending them both tumbling!</span>", "<span class='userdanger'>You land a passable [tackle_word] on [target], sending you both tumbling!</span>", target)
 			to_chat(target, "<span class='userdanger'>[user] lands a passable [tackle_word] on you, sending you both tumbling!</span>")
 
-			target.adjustStaminaLoss(stamina_cost)
+			target.adjustStaminaLoss(stamina_cost * 1.5)
 			target.Paralyze(5)
 			user.Knockdown(20)
-			target.Knockdown(25)
+			target.Knockdown(30)
```
Comment (silicons) [inline code/datums/components/tackle.dm:177]: hard knockdowns are kinda shitty

File: code/datums/components/tackle.dm [hunk 4]
```diff
@@ -163,24 +163,24 @@
 		if(-4 to -2) // glancing blow at best
 			user.visible_message("<span class='warning'>[user] lands a weak [tackle_word] on [target], briefly knocking [target.p_them()] off-balance!</span>", "<span class='userdanger'>You land a weak [tackle_word] on [target], briefly knocking [target.p_them()] off-balance!</span>", target)
 			to_chat(target, "<span class='userdanger'>[user] lands a weak [tackle_word] on you, briefly knocking you off-balance!</span>")
-
 			user.Knockdown(30)
+			target.adjustStaminaLoss(30)
```
Comment (silicons) [inline code/datums/components/tackle.dm]: 15

File: code/datums/components/tackle.dm [hunk 5]
```diff
@@ -205,9 +205,9 @@
 			user.SetKnockdown(0)
 			user.set_resting(FALSE, TRUE, FALSE)
 			user.forceMove(get_turf(target))
-			target.adjustStaminaLoss(40)
-			target.Paralyze(5)
-			target.Knockdown(30)
+			target.adjustStaminaLoss(80)
```
Comment (silicons) [inline code/datums/components/tackle.dm]: 65

File: code/datums/components/tackle.dm [hunk 6]
```diff
@@ -192,9 +192,9 @@
 			user.SetKnockdown(0)
 			user.set_resting(FALSE, TRUE, FALSE)
 			user.forceMove(get_turf(target))
-			target.adjustStaminaLoss(40)
-			target.Paralyze(5)
-			target.Knockdown(30)
+			target.adjustStaminaLoss(50)
```
Comment (kevinz000) [inline code/datums/components/tackle.dm:195]: 45

File: code/datums/components/tackle.dm [hunk 7]
```diff
@@ -205,9 +205,9 @@
 			user.SetKnockdown(0)
 			user.set_resting(FALSE, TRUE, FALSE)
 			user.forceMove(get_turf(target))
-			target.adjustStaminaLoss(40)
-			target.Paralyze(5)
-			target.Knockdown(30)
+			target.adjustStaminaLoss(80)
+			target.Paralyze(10) 
+			target.Knockdown(50)
```
Comment (silicons) [inline code/datums/components/tackle.dm]: hard knockdowns are bad

**Comment by silicons** [comment]
@kappa-sama yeah i thought that too until someone came along giving themselves gigantism and combat gloves every round

File: code/datums/components/tackle.dm (hunk 3 above)
Comment (Arturlang) [reply code/datums/components/tackle.dm:177]: Replace knockdowns with standard knockdowns? Do they still do the thing where you cant get up instantly and run off?

File: code/datums/components/tackle.dm (hunk 1 above)
Comment (Arturlang) [reply code/datums/components/tackle.dm]: Yeah but like, you can essentially escape it from it instantly invalidating the grab completely

**Review by Ghommie — APPROVED** [review]

**Pull request merged**